tests/test.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Test Kibana role
+  hosts: all
+
+  pre_tasks:
+    - name: Update apt
+      become: yes
+      apt:
+        cache_valid_time: 1800
+        update_cache: yes
+      tags:
+        - build
+
+  # The env vars allow Travis to easily override the versions
+  roles:
+    - role: sansible.elasticsearch
+      elasticsearch:
+        family: "{{ ( lookup('env','ES_FAMILY') == '' ) | ternary('1.7', lookup('env','ES_FAMILY')) }}"
```

Comment:
That's what I thought, but lookup env returns '' when it doesn't find the var, default needs false or none in order to trigger, it's a bug or at least inconsistent behaviour in the Jinja2 filter I think.